requests-for-comments/marketing-and-content-team.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Marketing & Content Team Questions
+
+As per [my PR on the handbook](https://github.com/PostHog/posthog.com/pull/6010) the Marketing team is potentially evolving to take greater ownership of all content, specifically working with Website & Docs to ensure the docs are up-to-date and maintained to a high-standard.
+
+This poses a bunch of questions about the team, who does what, and how we balance this work against existing marketing work. Here are some questions and answers in no particular order (please add your own):
+
+## Is this a desirable change?
+
+The argument in favor is we have the expertise and process in the content team already. Ian and Lior are both technically knowledgable and excellent writers, and I have opinions about en dashes 😅. 
+
+Another strong argument in favor is that the quality of the docs, and the ease with which users can find answers, is a strong driver of word-of-mouth growth. Given we have the skills and knowledge to contribute, doing so actively supports marketing team objectives.
+
+The risk is combining the two content flows means one or the other suffers, or forces people to spend more time on content they perhaps don't enjoy writing.
+
+For me, I'd imagine an 90/10 - split – i.e. 90% of time spent on existing markeitng content work, 10% on improving docs – but this might be more like 60/40 or 70/30 etc. for other members of the team, and wouldn't be a constant.
+
+## How should / should we divide docs responsibilities?
+
+One option would be to divide docs up among the team – e.g. Ian owns feature flags, Lior owns session replay and so on.
```

Comment:
How about the small team owner is responsible for making sure:
1. There aren't any large gaps for that product
2. New features are sufficiently documented
They can do both of these without actually writing the words themselves.

Reply:
This helps the main docs writers stay up to date, takes a little off their plate.

Reply:
There are also some docs that don't have teams like client libraries, infrastructure, etc.

Reply:
Yep, just posted in another comment but I see it being reasonable to work with (ie "nag?") relevant engineers to have them write docs where they have significantly more domain knowledge.